apps/web/src/actions/sdk/runDocumentAction.ts

```diff
@@ -28,13 +28,12 @@ export async function runDocumentAction({
   commitUuid,
   parameters,
 }: RunDocumentActionProps) {
-  const result = await getLatitudeApiKey()
-  if (result.error) return result
```

Comment:
Is better to return `Result`s from services. Then in the API endpoints or actions we call `unwrap`. But if for some reason we want to use this service inside another service we would had to deal with `try/catch` which is discusting

Reply:
But this is a server action, not a service

Reply:
Then me callo 😶